### PR TITLE
[ES] added content/es/stateful-apps.md

### LINKED ISSUE #1776: [Spanish] localize 'Stateful Apps'
Localize the `content/en/stateful-apps.md` document to Spanish

### PR DESCRIPTION
**Pull request by marianogg9**
### Describe your changes
- added `content/es/stateful-apps.md`.

### Related issue number or link (ex: `resolves #issue-number`)
- resolves #1776 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/stateful-apps.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: Aplicaciones con estado
+status: Completed
+category: Concepto
+tags: ["fundamental", "application", ""]
+---
+
+## ¿Qué es?
+
+Cuando hablamos de aplicaciones con estado (y [sin estado](/stateless-apps/)),
+ese estado refiere a cualquier dato que la aplicación necesita almacenar para funcionar.
```
Comment (ramrodo) [inline content/es/stateful-apps.md:11]: Not blocking suggestion, just a personal improving:

```suggestion
éste estado se refiere a cualquier dato que la aplicación necesita almacenar para funcionar.
```

File: content/es/stateful-apps.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Por ejemplo, cualquier tipo de sitio de ventas en línea que recuerda tu carro de compras es una aplicación con estado.
```
Comment (ramrodo) [inline content/es/stateful-apps.md:12]: No es una sugerencia obligada o bloqueante, solo una sugerencia personal que creo que se podría entender mejor al leer:

```suggestion
Por ejemplo, cualquier tipo de tienda en línea que recuerda tu carro de compras, es una aplicación con estado.
```

**Review by ramrodo — COMMENTED** [review]
Some optional suggestions.

@marianogg9 could you add the word **resolves** before the issue link (in the PR description), please? This will be helpful to close the issue related once this PR is approved and merged. Thanks :)

**Review by 92nqb — APPROVED** [review]

**Review by ramrodo — APPROVED** [review]
Thanks! LGTM

**Pull request merged**